### ISSUE TITLE
Full resync is not removing VETH interfaces in vpp-agent v2.5.0

### ISSUE THREAD

**Issue by artem-belov** [ligato/vpp-agent#1575]
After vpp-agent restarted I am recreating connections and catch:
```
4. CREATE [FAILED]:
          - key: config/linux/interfaces/v2/interface/SRC-1-veth
          - value: { name:"SRC-1-veth" type:VETH host_if_name:"SRC-1-veth" enabled:true veth:<peer_if_name:"SRC-1" >  } 
          - error: file exists
```
I was trying to find cause of the issue, and figured out that issue is in ed4177d28120bb8382b5812d9f99971b5df5cb41 
If I am reverting that commit, Full Resync removing old interfaces and everything works properly:
```
+======================================================================================================================+
| #0 - NB Transaction                                                                                       FullResync |
+======================================================================================================================+
  * transaction arguments:
      - seqNum: 0
      - type: NB Transaction, Full Resync
      - values:
          - key: config/linux/interfaces/v2/interface/DST-1-veth-MISSING_PEER
            val: <NIL>
          - key: config/linux/interfaces/v2/interface/SRC-1-veth-MISSING_PEER
            val: <NIL>

o----------------------------------------------------------------------------------------------------------------------o
  * executed operations (2019-12-03 05:13:09.239 +0000 UTC -> 2019-12-03 05:13:09.339 +0000 UTC, dur: 100ms):
      1. DELETE [PROPERTY DISCOVERED]:
          - key: linux/interface/state/DST-1-veth-MISSING_PEER/UP
          - value: <EMPTY> 
      2. DELETE [DISCOVERED]:
          - key: config/linux/interfaces/v2/interface/DST-1-veth-MISSING_PEER
          - value: { name:"DST-1-veth-MISSING_PEER" type:VETH host_if_name:"DST-1-veth" enabled:true phys_address:"46:7d:75:40:67:51" 
mtu:1500 veth:/* nil */  } 
      3. DELETE [PROPERTY DISCOVERED]:
          - key: linux/interface/state/SRC-1-veth-MISSING_PEER/UP
          - value: <EMPTY> 
      4. DELETE [DISCOVERED]:
          - key: config/linux/interfaces/v2/interface/SRC-1-veth-MISSING_PEER
          - value: { name:"SRC-1-veth-MISSING_PEER" type:VETH host_if_name:"SRC-1-veth" enabled:true phys_address:"46:c4:0b:6a:d1:79" 
mtu:1500 veth:/* nil */  } 
x----------------------------------------------------------------------------------------------------------------------x
| #0 - NB Transaction                                                                                      took 99.9ms |
x----------------------------------------------------------------------------------------------------------------------x
```

Full broken log:
[nsmd-forwarder-recovered-0-vppagent-forwarder.log](https://github.com/ligato/vpp-agent/files/3920393/nsmd-forwarder-recovered-0-vppagent-forwarder.log)

**Comment by artem-belov** [comment]
Log received using branch [master-errors](https://github.com/ligato/vpp-agent/tree/master-errors):
[vppagent-forwarder-master-errors.log](https://github.com/ligato/vpp-agent/files/3925235/vppagent-forwarder-master-errors.log)

**Comment by artem-belov** [comment]
[Log from vpp-agent v2.3.0](https://github.com/ligato/vpp-agent/files/3926391/vppagent-forwarder-v2.3.0.log)

**Comment by ondrej-fabry** [comment]
Possible fix https://github.com/ligato/vpp-agent/pull/1577

**Comment by ondrej-fabry** [comment]
Fix merged. Will be released as `2.5.1`.
